### PR TITLE
Fix some broken blog links

### LINKED ISSUE #2239: Please fix the link for the referenced blog post
Can you please update the link for the Blog Post?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: af0b8bf0-8cdb-e5ae-0344-e2e281f41969
* Version Independent ID: 90d5f106-4bee-1075-3e43-054556a301df
* Content: [Handling transaction commit failures - EF6](https://docs.microsoft.com/en-us/ef/ef6/fundamentals/connection-resiliency/commit-failures?source=docs#feedback)
* Content Source: [entity-framework/ef6/fundamentals/connection-resiliency/commit-failures.md](https://github.com/dotnet/EntityFramework.Docs/blob/master/entity-framework/ef6/fundamentals/connection-resiliency/commit-failures.md)
* Product: **entity-framework**
* Technology: **entity-framework-6**
* GitHub Login: @divega
* Microsoft Alias: **avickers**

### PR DESCRIPTION
**Pull request by ErikEJ**
fixes #2239

### CODE REVIEW

**Comment by ajcvickers** [comment]
Thanks Erik!

File: entity-framework/ef6/fundamentals/testing/testability-article.md [hunk 1]
```diff
@@ -958,10 +958,10 @@ In this paper we’ve demonstrated several approaches to creating testable code
 -   Robert C. Martin, “ [The Single Responsibility Principle](https://www.objectmentor.com/resources/articles/srp.pdf)”
 -   Martin Fowler, [Catalog of Patterns](https://www.martinfowler.com/eaaCatalog/index.html) from *Patterns of Enterprise Application Architecture*
 -   Griffin Caprio, “ [Dependency Injection](https://msdn.microsoft.com/magazine/cc163739.aspx)”
--   Data Programmability Blog, “ [Walkthrough: Test Driven Development with the Entity Framework 4.0](https://blogs.msdn.com/adonet/pages/walkthrough-test-driven-development-with-the-entity-framework-4-0.aspx)”.
--   Data Programmability Blog, “ [Using Repository and Unit of Work patterns with Entity Framework 4.0](https://blogs.msdn.com/adonet/archive/2009/06/16/using-repository-and-unit-of-work-patterns-with-entity-framework-4-0.aspx)”
+-   Data Programmability Blog, “ [Walkthrough: Test Driven Development with the Entity Framework 4.0](https://docs.microsoft.com/en-us/archive/blogs/adonet/walkthrough-test-driven-development-with-the-entity-framework-4-0)”.
```
Comment (smitpatel) [inline entity-framework/ef6/fundamentals/testing/testability-article.md:961]: This causes warning due to en-us being in link.
See https://github.com/dotnet/EntityFramework.Docs/issues/1900

Comment (ErikEJ) [reply entity-framework/ef6/fundamentals/testing/testability-article.md:961]: Would you like a new PR?

Comment (ErikEJ) [reply entity-framework/ef6/fundamentals/testing/testability-article.md:961]: PR here: https://github.com/dotnet/EntityFramework.Docs/pull/2264

**Pull request merged**